packages/web-react/src/components/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type VisuallyHiddenProps = {
+  children: string;
+  elementType?: 'span' | 'div' | 'legend';
+};
+
+const defaultProps = {
+  elementType: 'span',
+};
+
+const VisuallyHidden = (props: VisuallyHiddenProps) => {
```

Comment:
@adamkudrna @crishpeen @pavelklibani Had you a discussion about whether this component could also support style props and transferring props or not? Because I am missing these two features here. About transferring props I am quite sure that the feature should be here but about escape hatches of styling, I am not.

Reply:
I would add both. I can imagine you might need escape hatch to eg. add a classname to show the text in print and so on. And we need transfer props too.

Reply:
This has been fixed.